source/examples/rapids-sagemaker-hpo/entrypoint.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-source activate rapids
```

Comment:
There is no `rapids` conda environment in the `rapidsai/base` images (which this entrypoint runs in).

They put everything in `base`.